### ISSUE TITLE
Android TextField A11y

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16392]
- [ ] touch target for prefixIcon/suffixIcon of InputDecoration is too small (should be 48dp, is 24dp)
- [ ] hint is not read out when textfield is toggled to edit mode

**Comment by goderbauer** [comment]
This bug has been split in two: https://github.com/flutter/flutter/issues/16672 and https://github.com/flutter/flutter/issues/16673
